src/Microsoft.ML.FastTree/FastTree.cs

```diff
@@ -2988,37 +2988,10 @@ void ICanSaveInIniFormat.SaveAsIni(TextWriter writer, RoleMappedSchema schema, I
             Host.CheckValue(writer, nameof(writer));
             Host.CheckValue(schema, nameof(schema));
```

Comment:
Don't need to check here as you don't use it in this function.

Reply:
Removed.

---
In reply to: [243139197](https://github.com/dotnet/machinelearning/pull/1929#discussion_r243139197) [](ancestors = 243139197)